### DISCUSSION TITLE
Install addon in docker

### DISCUSSION THREAD

**Discussion by WueBumbum** [homeassistant-apps/app-cloudflared#672] [Ideas]
Hello,

it is nearly 2 years since  install issue "docker-compose #136" was closed due not planned. Have the plans changed in the meantime?

thank you and best regards
Bumbum

**Comment by elcajon** [comment]
It is simply not possible to run the add-on with docker-compose as the add-on requires the home-assistant supervisor which is only available (and supported) with HAOS and HA supervised installations.
But you can just use the official cloudflare/cloudflared docker images in your docker-compose files.

**Comment by WueBumbum** [comment]
I just started tinkering with HA this weekend and am new to this stuff. You explanation makes sense. May I ask one more question? Do I have to install the Cloudflared Docker Container within my Home Assistant Container or in a new Container to achieve the Cloudflared-tunnel in HA?

**Comment by brenner-tobias** [comment]
If you are using a local docker-compose set-up, simply use the official [cloudflared docker image ](https://hub.docker.com/r/cloudflare/cloudflared) as a separate container.
Let me know if there are any additional questions.
